terraform/pulsar-mel3/pulsar-mel3-initial.tf

```diff
@@ -1,7 +1,7 @@
 # Pulsar-mel3
 resource "openstack_compute_instance_v2" "pulsar-mel3" {
   name            = "pulsar-mel3"
-  image_name      = "NeCTAR Ubuntu 20.04 LTS (Focal) amd64"
+  image_name      = "NeCTAR Ubuntu 20.04 LTS (Focal) amd64 [v9]"
```

Comment:
It may be better to go with the image id here. But the versioned name is probably ok too

Reply:
agreed, image id is better because it won't go out of date